model_utils/tracker.py

```diff
@@ -240,21 +289,17 @@ def _patch(self, model, method, fields_kwarg):
 
         @wraps(original)
         def inner(instance, *args, **kwargs):
-            ret = original(instance, *args, **kwargs)
             update_fields = kwargs.get(fields_kwarg)
-            if not update_fields and update_fields is not None:  # () or []
-                fields = update_fields
-            elif update_fields is None:
-                fields = None
+            if update_fields is None:
+                fields = self.fields
```

Comment:
Nice you took that improvement from my PR ;-)

Reply:
Yep, it's much more simple to understand.